### PR TITLE
Tweak wording of activate prompt

### PR DESCRIPTION
**Pull request by hadley**
I think this makes it more clear what happens if you say no.

### CODE REVIEW

File: R/activate.R [hunk 1]
```diff
@@ -149,16 +149,15 @@ renv_activate_prompt <- function(action, library, prompt, project) {
...
+    "See `?renv::activate` for more details.",
     ""
   )
-
   notice <- sprintf(fmt, action)
   vwritef(notice)
 
-  fmt <- "Would you like to activate this project before %s() is called?"
-  question <- sprintf(fmt, action)
+  question <- "Would you like to activate the project now?"
```
Comment (kevinushey) [inline R/activate.R:160]:
```suggestion
  question <- "Would you like to activate this project now?"
```

File: R/activate.R [hunk 2]
```diff
@@ -149,16 +149,15 @@ renv_activate_prompt <- function(action, library, prompt, project) {
   fmt <- lines(
     "",
     "This project has not yet been activated.",
-    "Activating this project will ensure the project library is used when %s() is called.",
-    "Please see `?renv::activate` for more details.",
+    "This means that %s() will modify your global library.",
```
Comment (kevinushey) [inline R/activate.R:152]: This line would only be true if this were called in response to restore / clean; if we kept the snapshot prompt then it wouldn't be accurate (so I guess this PR depends on https://github.com/rstudio/renv/pull/1222).

**Review by kevinushey — COMMENTED** [review]
I feel like we can still find some better text to use here. We might want to just have different tailored messages for the different activation scenarios (restore / clean / ...)

**Comment by hadley** [comment]
Alternatively, maybe it would be better to error if the user declines to activate? It doesn't seem like we want folks using `restore()` to update the system libraries?

**Comment by hadley** [comment]
If we merge #1243, the default prompt will become yes, we could easily make error if the user declines interatively and succeed if run non-interactively.  (But I do think it makes renv harder to understand if actions sometimes change the system library depending on what else is going on.)

**Comment by hadley** [comment]
I've had another stab at this with all the recent changes, but I think the problem is that a Y/N just isn't quite enough. I think the message should really be something like:

```
This project has not yet been activated.
What do you want to do?

1. Activate the project and set up a project library.
2. Continue and use the global library
3. Cancel
```

**Comment by kevinushey** [comment]
I like this idea as well -- do you want to file a separate issue to capture this, or just submit a new PR?

**Comment by hadley** [comment]
I'll just make a new PR.

**Pull request closed without merging**